docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,211 @@
+load testing tool.
+A prebuilt container is also available on [Docker Hub](https://hub.docker.com/r/redislabs/memtier_benchmark/)
+Although Redis offers the `redis-benchmark` tool in its repository, it has not been as efficient
+as `memtier_benchmark` and it often becomes the bottleneck instead of Dragonfly.
+
+We also developed our own tool [dfly_bench](https://github.com/dragonflydb/dragonfly/blob/main/src/server/dfly_bench.cc),
+which can be built from source in the Dragonfly repository.
+
+
+## Methodology
+- **Remote Deployment:** Dragonfly is a multi-threaded server designed to run remotely.
+Therefore, we recomment running the load testing client and server on separate machines for a more accurate representation of real-world performance.
+- **Minimizing Latency:** Locate client and server within the same Availability Zone and use private IPs for optimal network performance. If you benchmark in the AWS cloud, consider an AWS Cluster placement group
+for the lowest possible latency. The rationale behind this - to remove any environmental factors
+that might skew the test results.
+- **Server vs. Client Resources**: Use a more powerful instance for the load testing client
+to avoid client-side bottlenecks.
+
+The remainder of this document will discuss how to set up a benchmark in the AWS cloud
+to observe millions of QPS from a single instance.
+
+## Load testing configuration
+We used Dragonfly v1.15.0 (latest at the time of writing) with the following arguments:
+`./dragonfly --logtostderr  --dbfilename=`
+
+Please notice that Dragonfly uses of all the available vCPUs by default on the server.
+If you want to control explicitly number of threads in Dragonfly you can add `--proactor_threads=<N>`.
+Both the client and server instances run `Ubuntu 23.04` OS with kernel version 6.2.
+In line with our recommendations above, we used internal IPs for connecting and
+used stronger `c7gn.16xlarge` instance with 64 vCPUs for the load-testing program.
+
+## Dragongly on `c6gn.12xlarge`
+
+### Write-only test
+On the loadtest instance (c7gn.16xlarge with 64 vCPUs) we run:
+`memtier_benchmark -s $SERVER_PRIVATE_IP --distinct-client-seed --hide-histogram --ratio 1:0 -t 60 -c 20 -n 200000`
+
+The run ended with the following summary:
+
+```
+============================================================================================================================
+Type         Ops/sec     Hits/sec   Misses/sec    Avg. Latency     p50 Latency     p99 Latency   p99.9 Latency       KB/sec
+----------------------------------------------------------------------------------------------------------------------------
+Sets      4195628.23          ---          ---         0.39283         0.37500         0.68700         2.54300    323231.06
+
+```
+
+In this test, we reached almost 4.2M queries per second (QPS) with the average latency of
+392us between the `memtier_benchmark` and Dragonfly. Consequently, the P50 latency was 375us, P99 - 687us
```

Comment:
```suggestion
392us (0.4ms) between the `memtier_benchmark` and Dragonfly. Consequently, the P50 latency was 375us (0.4ms), P99 - 687us (0.7ms)
```
It might be too much to add in all places, but at least for the first one it emphasizes the extreme low latency (and maybe also reduces confusion - "hundreds" sound like a lot)